apps/web/middleware.ts

```diff
@@ -39,6 +39,10 @@ const middleware: NextMiddleware = async (req) => {
   return NextResponse.next();
 };
 
+export const config = {
+  matcher: ["/api/collect-events/:path*", "/api/auth/:path*", "/apps/routing_forms/:path*", "/:path*/embed"],
```

Comment:
where is /embed path used for?

Reply:
cc: @hariombalhara

Reply:
@Udit-takkar Does it fix the issue for you? I will also test locally.

Reply:
<img width="753" alt="Screenshot 2022-11-01 at 5 14 37 PM" src="https://user-images.githubusercontent.com/1780212/199225485-6e500823-bd10-442f-943e-05b68671f2df.png">

Reply:
yeah i tested locally . the middleware only executed for these paths. just test the embed url once

Reply:
I could replicate locally that in main, updateProfile request is stuck and in this branch it immediately completes

Reply:
@Udit-takkar Right now it's possible to be in a situation where someone adds logic in middleware fn to do something for a URL, but the logic never executes. This is because there is a separate array governing which URLs can invoke middleware.
@emrysal  and  I were thinking how to solve this. Can you try something like this and see if it solves the problem and still let us use middleware the current way. 
![image](https://user-images.githubusercontent.com/1780212/199238316-47049c4b-c032-467d-8f1e-7a5918554a59.png)

Reply:
@emrysal shared and I agree that this isn't something that we need right now and makes things complex. So, we can skip it for now. 
Just ignore the last comment @Udit-takkar